systems/plants/constraint/constructPtrRigidBodyConstraintmex.cpp

```diff
@@ -2,6 +2,7 @@
 #include "RigidBodyManipulator.h"
 #include "RigidBodyConstraint.h"
 #include "drakeUtil.h"
+#include "controlUtil.h"
```

Comment:
when I did the controller reorg, I created a `sizecheck()` and put it in `controlUtil`. We can certainly move it elsewhere